tests/unit/specs/components/common/__snapshots__/GoogleAuthButton.spec.js.snap

```diff
@@ -8,7 +8,7 @@ exports[`GoogleAuthButton render with gapi should correctly render GoogleAuthBut
 `;
 
 exports[`GoogleAuthButton render without gapi should correctly render GoogleAuthButton component empty if auth2 isn't loaded 1`] = `
-<v-button-stub skin="social" isloading="true" size="normal" disabled="true" type="button" data-test="submit-button-google">
```

Comment:
Is it ok that we don’t have loading?

Reply:
yep, because we have disabled=true. this is initial state
hmm, maybe we should pass `is-loading=true` with `disabled=true` for show user, that button can't be clicked, but it's in loading state for a while?

Reply:
Passing isLoading will be good for user

Reply:
done